### ISSUE TITLE
resourceName is empty string

### ISSUE THREAD

**Issue by tritonas00** [RigsOfRods/rigs-of-rods#2410]
This issue happened on multiplayer just now and a few times in the past. There was only one user in the neo server, that caused this error to other clients who were trying to join the server:

```
RuntimeAssertionException: resourceName is empty string in resourceExistsInAnyGroupImpl at /home/babis/Downloads/ror-dependencies/Source/ogre/OgreMain/src/OgreResourceGroupManager.cpp (line 1642)
```

As long as the user was in i was getting the error. I immediately removed all my mods except the terrain and tried again, because i thought something he spawned was causing this. The error still happened! Once i kicked him after warning him many times, the error gone.

@only-a-ptr how is this possible? Without **any** mod in my mods dir...

The RoRBot complained that the truckinfo `type` was `unknown`
https://github.com/CuriousMike56/RoRServerBot/blob/master/RoR_client.py#L490
https://github.com/CuriousMike56/RoRServerBot/blob/master/RoR_client.py#L198

the actor he spawned had no file extension?

also ror could not identify his username after a while, reported it like `unknown 0`, maybe thats the issue

**Comment by only-a-ptr** [comment]
Do you have a server log?
The message shows it must have been one of these places: https://github.com/RigsOfRods/rigs-of-rods/search?q=resourceExistsInAnyGroup
Could have been a .fixed or .machine on the terrain, or attempt to find caelum config. I doubt those guys enabled the experimental flexbody cache.
Anyway that error should be non-fatal.

**Comment by CuriousMike56** [comment]
> @only-a-ptr  I doubt those guys enabled the experimental flexbody cache.

You'd be surprised - a good amount of the logs I've seen while providing support had it enabled.

**Comment by tritonas00** [comment]
> Do you have a server log?
No unfortunately

I suggest keep the error on log and just proceed on those cases

**Comment by CuriousMike56** [comment]
https://github.com/RigsOfRods/ror-server/issues/96#issuecomment-538607923

**Comment by only-a-ptr** [comment]
Resolved by #2560
